src/components/Common/JobLink.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Flex from 'styled-flex-component'
+import styled from 'styled-components'
+
+import ExternalAnchor from '../Common/ExternalAnchor'
+import { BodyPrimary } from '../Typography'
+import Hr from '../Common/Hr'
+
+const StyledExternalAnchor = styled(ExternalAnchor)`
+  height: 107px;
```

Comment:
is this needed? shouldn't it be a min-height? (Im afraid of content being cut).
Also, we should use `remcalc`?

Reply:
Yeah good point. I was fixing the height cos that's in the specs:
![image](https://user-images.githubusercontent.com/15656538/56035892-54d51500-5d23-11e9-95f0-8c3903433031.png)

But this is a lazy half-done implementation cos I'm rushing. I'll put it back to how it was cos I don't think I can add ellipses, or some other solution to cut of the text, before I leave.